### PR TITLE
Playbook Generation wizardId support

### PR DESCRIPTION
**Pull request by TamiTakamiya**
Support wizardId for Playbook Generation feature to provide a unique identifier that belong to a specific Playbook Generation UI session.  

This also changed the behavior of generationId, which was re-used for multiple the generations API calls from a single Playbook Generation UI session, not to be re-used, i.e. always a unique ID will be provided for each generations API call.

### CODE REVIEW

File: test/ui-test/lightspeedUiTest.ts [hunk 1]
```diff
@@ -590,7 +590,7 @@ export function lightspeedUIAssetsTest(): void {
       const notification = notifications[0];
       expect(await notification.getMessage()).equals(
         "Enable lightspeed services from settings to use the feature.",
-      ).to.be.true;
```
Comment (TamiTakamiya) [inline test/ui-test/lightspeedUiTest.ts:593]: This is not directly related to the wizardId support, but was found when running the UI tests for verifying this PR.  I am not sure how it was successfully completed before...

**Review by goneri — APPROVED** [review]
